monitoring/google/cloud/monitoring_v3/gapic/enums.py

```diff
@@ -332,56 +392,6 @@ class Reducer(enum.IntEnum):
         REDUCE_PERCENTILE_05 = 12
 
 
-class NotificationChannel(object):
-    class VerificationStatus(enum.IntEnum):
-        """
-        Indicates whether the channel has been verified or not. It is illegal
-        to specify this field in a
-        ````CreateNotificationChannel````
-        or an
-        ````UpdateNotificationChannel````
-        operation.
-
-        Attributes:
-          VERIFICATION_STATUS_UNSPECIFIED (int): Sentinel value used to indicate that the state is unknown, omitted, or
-          is not applicable (as in the case of channels that neither support
-          nor require verification in order to function).
-          UNVERIFIED (int): The channel has yet to be verified and requires verification to function.
-          Note that this state also applies to the case where the verification
-          process has been initiated by sending a verification code but where
-          the verification code has not been submitted to complete the process.
-          VERIFIED (int): It has been proven that notifications can be received on this
-          notification channel and that someone on the project has access
-          to messages that are delivered to that channel.
-        """
-        VERIFICATION_STATUS_UNSPECIFIED = 0
-        UNVERIFIED = 1
-        VERIFIED = 2
-
-
-class AlertPolicy(object):
-    class ConditionCombinerType(enum.IntEnum):
-        """
-        Operators for combining conditions.
-
-        Attributes:
-          COMBINE_UNSPECIFIED (int): An unspecified combiner.
-          AND (int): Combine conditions using the logical ``AND`` operator. An
-          incident is created only if all conditions are met
-          simultaneously. This combiner is satisfied if all conditions are
-          met, even if they are met on completely different resources.
-          OR (int): Combine conditions using the logical ``OR`` operator. An incident
-          is created if any of the listed conditions is met.
-          AND_WITH_MATCHING_RESOURCE (int): Combine conditions using logical ``AND`` operator, but unlike the regular
-          ``AND`` option, an incident is created only if all conditions are met
-          simultaneously on at least one resource.
-        """
-        COMBINE_UNSPECIFIED = 0
-        AND = 1
-        OR = 2
-        AND_WITH_MATCHING_RESOURCE = 3
-
-
```

Comment:
Although the enum-order bug (https://github.com/googleapis/gapic-generator/issues/2319) does show up here, there are other substantive changes.